### PR TITLE
Add qredis recipe

### PR DESCRIPTION
**Pull request by tiagocoutinho**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by goanpeca** [comment]
Thanks for the recipe @tiagocoutinho 

The linux build has the following issue when running the recipe tests:


```
+ qredis --help
Traceback (most recent call last):
  File "/home/conda/staged-recipes/build_artifacts/qredis_1629195414237/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/bin/qredis", line 6, in <module>
    from qredis.window import main
  File "/home/conda/staged-recipes/build_artifacts/qredis_1629195414237/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib/python3.9/site-packages/qredis/window.py", line 5, in <module>
    from qtpy.QtGui import QIcon
  File "/home/conda/staged-recipes/build_artifacts/qredis_1629195414237/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib/python3.9/site-packages/qtpy/QtGui.py", line 21, in <module>
    from PyQt5.QtGui import *
ImportError: libGL.so.1: cannot open shared object file: No such file or directory
Tests failed for qredis-1.0.0-pyhd8ed1ab_0.tar.bz2 - moving package to /home/conda/staged-recipes/build_artifacts/broken
WARNING conda_build.build:tests_failed(2955): Tests failed for qredis-1.0.0-pyhd8ed1ab_0.tar.bz2 - moving package to /home/conda/staged-recipes/build_artifacts/broken
```

**Comment by tiagocoutinho** [comment]
Thanks for reporting @goanpeca ,

It looks like for applications that need PyQt/Qt stack, the docker image used for the linux builds is insufficient. 
Do you have any recommendation on how to fix this?
I guess I am asking for some help here.

Thanks in advance

**Comment by tiagocoutinho** [comment]
I found that some recipes have this `yum_requirements.txt` file with contents:
```
 mesa-libGL
```

I will try with this

File: recipes/qredis/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+  run:
+    - msgpack-python
+    - msgpack-numpy
+    - qtpy
+    - pyqt =5
```
Comment (goanpeca) [inline recipes/qredis/meta.yaml:28]: Any reason to not use `>=5` ?

Comment (tiagocoutinho) [reply recipes/qredis/meta.yaml:28]: Yes, qredis direct dependency is qtpy. So far qtpy only supports PyQt4 or PyQt5 but not PyQt6.
Would it make more sense to put `<6` instead ?

Comment (goanpeca) [reply recipes/qredis/meta.yaml:28]: That would probably be more correct in the face of `only supports PyQt4 or PyQt5`, but I guess `pyqt =5` works just fine

Also making a comment https://github.com/spyder-ide/qtpy/issues/233 here could help ;-)

Comment (tiagocoutinho) [reply recipes/qredis/meta.yaml]: Sorry for the silence. I have no excuse.
Looks like qtpy>=2 added support for PyQt6 so I have just updated `meta.yaml`.

**Comment by carterbox** [comment]
@conda-forge-admin, please restart ci

**Pull request merged**